packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -124,6 +124,13 @@ function getConfigFileMigrationsFactory(homeDir, defaultConfigs) {
           });
         return configFile;
       },
+      '0.25.0': (configFile) => {
```

Comment:
Is this the right way to handle migrations for `-dev` releases?

Reply:
You can set 0.25.0-dev.18